addons/rating/views/rating_templates.xml

```diff
@@ -4,12 +4,14 @@
         <template id="rating_external_page_view" name="Rating Page Done">
             <t t-call="web.frontend_layout">
                 <div class="container pt-5">
+                    <div class="oe_structure" id="oe_structure_rating_header_thanks_you"/>
```

Comment:
I didn't know the utily of this class the additional `id` here.
I guess it is to allow the user to edit the submit_feedback page with website editor. 🤔 
So, we should perhaps only do this when we have `website` module instead of doing it directly in the `rating` module.

What do you think? 🤔

Reply:
Well it would be better to do it through an override indeed, but I don't think we have a "website_rating"  module anymore. So not sure we have a better option, adding a module just for that is maybe a bit too much ?